src/player_activity.cpp

```diff
@@ -1,6 +1,30 @@
+#include "advanced_inv.h"
+#include "debug.h"
+#include "game.h"
+#include "player.h"
 #include "player_activity.h"
+#include "veh_interact.h"
 #include "translations.h"
 
+#include <ostream>
+#include <sstream>
+
+
+extern game* g;
```

Comment:
Why repeat this here, it's already declared in game.h, which is included.

Another thing: funny indentation, sometimes two spaces, sometimes 4 spaces. Project standard is 4 spaces (no tabs), see doc/CODE_STYLE.txt